lib/iiif_manifest/v3/manifest_builder/canvas_builder_decorator.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# OVERRIDE IIIFManifest v1.3.1 to use the parent's title as the label instead of the filename
+
+module IIIFManifest
+  module V3
+    module ManifestBuilderDecorator
+      module CanvasBuilderDecorator
```

Comment:
TODO make sure this works and that we are generating and using v3 manifests everywhere.